sos/report/plugins/proxmox.py

```diff
@@ -0,0 +1,114 @@
+# Copyright (C) 2024 ORNESS/Ditrit Drien Breton <drien.breton@orness.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+import json
+from pathlib import Path
+
+from sos.report.plugins import Plugin, DebianPlugin, PluginOpt
+
+
+class Proxmox(Plugin, DebianPlugin):
+    """
+    This plugin will capture information about the system's
+    Proxmox Virtualization Environment.
+
+    It will collect information about the cluster, nodes, pools and
+    storage from the Proxmox API.
+    """
+    short_desc = 'Proxmox cluster information'
+
+    plugin_name = 'proxmox'
+    packages = ('proxmox-ve',)
+
+    option_list = [
+        PluginOpt('output_formats',
+                  desc='List of output formats to use '
+                       'for the commands separated by ":".',
+                  default='text',
+                  ),
+        PluginOpt('cmd_override_file',
+                  desc='File containing the commands '
+                       'to run instead of the default ones.',
+                  default=None,
+                  val_type=[None, Path, str],
+                  ),
+    ]
+
+    def setup(self):
+        output_formats = self.get_option('output_formats').split(':')
+
+        commands = [
+            'cluster/resources',
+            'cluster/config/nodes',
+            'cluster/options',
+            'nodes/:id/status',
+            'nodes/:id/storage',
+            'nodes/:id/network',
+            'pools',
+            'storage',
+            'storage/:sid',
+            'cluster/ceph/status'
+            'cluster/ceph/metadata'
+            'cluster/ceph/flags'
+        ]
+
+        cmd_paths = []
+
+        override_file = self.get_option('cmd_override_file')
+        if override_file is not None:
+            with open(override_file, 'r') as f:
```

Comment:
Could you please add `, encoding='UTF-8'`, likewise we are improving in all plugins (cf. e.g. https://github.com/sosreport/sos/pull/3546/files#diff-ded1fd003cd94bd33e0094c2e211a655c38c53a382badd42b0784a6bd0e37f83R136)?